core/src/main/java/io/questdb/log/LogRollingFileWriter.java

```diff
@@ -194,7 +194,7 @@ public void close() {
             Unsafe.free(buf, nBufferSize, MemoryTag.NATIVE_LOGGER);
             buf = 0;
         }
-        if (ff.closeChecked(this.fd)) {
+        if (ff.close(this.fd)) {
```

Comment:
I'd keep only this line change in this file. All other changes are not necessary considering #2935

Reply:
good point 👍